sdk/loadtesting/azure-developer-loadtesting/azure/developer/loadtesting/_operations/_patch.py

```diff
@@ -0,0 +1,446 @@
+        self._initial_response = None
+        self._polling_interval = interval
+        self._status = None
+        self._termination_statuses = ["VALIDATION_SUCCESS", "VALIDATION_FAILED", "VALIDATION_NOT_REQUIRED"]
+
+    def _update_status(self) -> None:
+        self._status = self._resource["validationStatus"]
+
+
+class TestRunStatusPoller(LoadTestingPollingMethod):
+    def __init__(self, interval=5) -> None:
+        self._resource = None
+        self._command = None
+        self._initial_response = None
+        self._polling_interval = interval
+        self._status = None
+        self._termination_statuses = ["DONE", "FAILED", "CANCELLED"]
+
+    def _update_status(self) -> None:
+        self._status = self._resource["status"]
+
+
+class LoadTestAdministrationClientOperationsMixin(AdministrationOperationsGenerated):
+    """
+    for performing the operations on the Administration Subclient
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(LoadTestAdministrationClientOperationsMixin, self).__init__(*args, **kwargs)
+
+    @distributed_trace
+    def begin_upload_test_file(
+        self, test_id: str, file_name: str, body: IO, *, file_type: Optional[str] = None, **kwargs: Any
+    ) -> LROPoller[JSON]:
+        """Upload file to the test
+
+        :param test_id: Unique id for the test
+        :type test_id: str
+        :param file_name: Name of the file to be uploaded
+        :type file_name: str
+        :param body: File content to be uploaded
+        :type body: IO
+        :param file_type: Type of the file to be uploaded
+        :type file_type: str
+        :return: An instance of LROPoller object to check the validation status of file
+        :rtype: ~azure.developer.loadtesting._polling.LoadTestingLROPoller
+        :raises ~azure.core.exceptions.HttpResponseError:
+        :raises ~azure.core.exceptions.ResourceNotFoundError:
+        """
+
+        polling_interval = kwargs.pop("_polling_interval", None)
+        if polling_interval is None:
+            polling_interval = 5
+        upload_test_file_operation = super().begin_upload_test_file(
+            test_id=test_id, file_name=file_name, body=body, file_type=file_type, **kwargs
+        )
+
+        command = partial(self.get_test_file, test_id=test_id, file_name=file_name)
+
+        create_validation_status_polling = ValidationCheckPoller(interval=polling_interval)
+        return LROPoller(command, upload_test_file_operation, lambda *_: None, create_validation_status_polling)
+
+
+class LoadTestRunClientOperationsMixin(TestRunOperationsGenerated):
+    """
+    class to perform operations on TestRun
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(LoadTestRunClientOperationsMixin, self).__init__(*args, **kwargs)
+
+    @overload
+    def begin_test_profile_run(
+        self, test_profile_run_id: str, body: _models.TestProfileRun, *,content_type: Optional[str] = None, **kwargs: Any
+    ) -> LROPoller[JSON]:
```

Comment:
[For SDK Team] : This should be `LROPoller[_models.TestProfileRun]` ?

Same for TestRun and their equivalents in the asyncio variants